src/MovieDetailView/MovieDetailView.js

```diff
@@ -12,23 +12,33 @@ const MovieDetailView = ({ selectedMovie }) => {
 
       <div className="detail-container">
         <div className="detail-left-container">
-         
+
           <img src={selectedMovie.poster_path} className="detail-poster"/>
         </div>
 
         <div className="detail-right-container">
-        <h1 className="detail-title">{selectedMovie.title}</h1>
-        <h3 className="detail-tagline">{selectedMovie.tagline}</h3>
-        <p className="overview">{selectedMovie.overview}</p>
-        <ul>
-          <li>Rating: {selectedMovie.average_rating.toFixed(1)}/10</li>
-          <li>Budget: ${selectedMovie.budget}</li>
-          <li>Revenue: ${selectedMovie.revenue}</li>
-          <li>Runtime: {selectedMovie.runtime}</li>
-          <li>Release Date: {selectedMovie.release_date}</li>
-          <li>Tags: {selectedMovie.genres.toString().split(',').join(', ')}</li>
-        </ul>
-      </div>
+          <h1 className="detail-title">{selectedMovie.title}</h1>
```

Comment:
Seeing lots of `selectedMovie._____` here - this could be a could usecase for destructuring!